app/src/main/java/com/kickstarter/viewmodels/ProjectSearchResultHolderViewModel.kt

```diff
@@ -50,18 +52,18 @@ interface ProjectSearchResultHolderViewModel {
         fun displayPrelaunchProjectBadge(): Observable<Boolean>
     }
 
-    class ViewModel(@NonNull environment: Environment) :
-        ActivityViewModel<ProjectSearchResultViewHolder>(environment), Inputs, Outputs {
```

Comment:
Same question here, instead of activityviewmodel which was our own design, why not just use a viewmodel extension?